.github/test-a-feature.md

```diff
@@ -103,4 +109,49 @@ yarn test
 yarn test:watch
 ```
 
-[1]: https://github.com/stardust-ui/react/tree/master/test/specs/commonTests
\ No newline at end of file
+[1]: https://github.com/stardust-ui/react/tree/master/test/specs/commonTests
+
+## Screener Tests
+
+For some components, it is necessary to write screenshot tests in order to check they render properly. For each component added to the docsite, a screenshot test is automatically created. This checks if that the component is rendered in a consistent way, as it checks the visual differences between the previous and the current rendering. We use [screener-io](https://screener.io/) to achieve this.
```

Comment:
`For each component` => `For each component _example_`

Reply:
thank you!